src/codewhisperer/activation.ts

```diff
@@ -421,10 +420,7 @@ export async function activate(context: ExtContext): Promise<void> {
                     }
                 }
 
-                CodeWhispererCodeCoverageTracker.getTracker(
-                    e.document.languageId,
-                    context.extensionContext.globalState
-                )?.countTotalTokens(e)
+                CodeWhispererCodeCoverageTracker.getTracker(e.document.languageId)?.countTotalTokens(e)
```

Comment:
`e.document.languageId` -> `runtimeLanguageContext.mapToRuntimeLanguage()` ?

e.document.languageId may contain jsx, javascript react

Reply:
yep, sounds good let me fix it real quick